src/hooks/useMarketsData.ts

```diff
@@ -128,6 +132,7 @@ export const useMarketsData = ({
   const sevenDaysSparklineData = usePerpetualMarketSparklines();
   const featureFlags = useAllStatsigGateValues();
   const unlaunchedMarkets = useMetadataService();
+  const hasMarketIds = Object.keys(allPerpetualMarkets).length > 0;
```

Comment:
this name doesn't feel quite right to me, I'm not quite sure what it means or is used for exactly.

Reply:
Can update, but the gist is that we want to know if the v4_markets channel was successfully subscribed to